AsyncDisplayKit/Layout/ASAbsoluteLayoutSpec.h

```diff
@@ -10,16 +10,33 @@
 
 #import <AsyncDisplayKit/ASLayoutSpec.h>
 
+/** How much space the spec will take up. */
+typedef NS_ENUM(NSInteger, ASAbsoluteLayoutSpecSizing) {
+  /** The spec will take up the maximum size possible. */
+  ASAbsoluteLayoutSpecSizingDefault,
+  /** Computes a size for the spec that is the union of all childrens' frames. */
+  ASAbsoluteLayoutSpecSizingSizeToFit,
+};
+
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- * A layout spec that positions children at fixed positions.
- * 
- * Computes a size that is the union of all childrens' frames.
+ A layout spec that positions children at fixed positions.
  */
 @interface ASAbsoluteLayoutSpec : ASLayoutSpec
 
 /**
+ How much space will the spec taken up
+ */
+@property (nonatomic, assign) ASAbsoluteLayoutSpecSizing sizing;
+
+/**
+ @param sizing How much space the spec will take up
+ @param children Children to be positioned at fixed positions
+ */
++ (instancetype)absoluteLayoutSpecWithSizing:(ASAbsoluteLayoutSpecSizing)sizing children:(NSArray<id<ASLayoutElement>> *)children AS_WARN_UNUSED_RESULT;
```

Comment:
All the other layout spec have the children at the end and I think it's the right approach also here.